src/java/fr/paris/lutece/portal/web/user/attribute/AttributeFieldJspBean.java

```diff
@@ -222,8 +222,7 @@ public String doModifyAttributeField( HttpServletRequest request )
                 return AdminMessageService.getMessageUrl( request, Messages.MANDATORY_FIELDS, AdminMessage.TYPE_STOP );
             }
 
-            AttributeField attributeField = new AttributeField( );
-            attributeField.setIdField( nIdField );
+            AttributeField attributeField = _attributeFieldService.getAttributeField( nIdField );
```

Comment:
This changes not only the position field, but also max_size_enter, width, and height. For now, let's just deal with position. In another patch, we can remove the useless fields